platform/android/java/app/build.gradle

```diff
@@ -95,6 +97,10 @@ android {
         targetSdkVersion getExportTargetSdkVersion()
     }
 
+    buildTypes {
+        release_debug {}
```

Comment:
What's the use case for the `release_debug` build type?

Reply:
I don't know about gradle integration specifically, but `release_debug` is the build type we use for release builds of the editor. It means "release (optimized) build with debugging/dev tools (`DEBUG_ENABLED`)".

The build matrix for desktop platforms is:

- Editor build: `tools=yes target=release_debug`
- Release template: `tools=no target=release`
- Debug template: `tools=no target=release_debug`

And then `target=debug` builds which are unoptimized and meant for engine developers (not part of the official builds matrix).

So for the Android editor we'll indeed want to use `target=release_debug`, i.e. the editor build is similar to a debug export template.